src/main/kotlin/org/rust/ide/intentions/DemorgansLawIntention.kt

```diff
@@ -0,0 +1,107 @@
+package org.rust.ide.intentions
+
+import com.intellij.codeInsight.intention.PsiElementBaseIntentionAction
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.tree.IElementType
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.util.elementType
+import org.rust.lang.core.psi.util.parentOfType
+
+class DemorgansLawIntention : PsiElementBaseIntentionAction() {
+    override fun getText() = "DeMorgan Law"
```

Comment:
yeah, fair enough, i overlooked this design first time 😂 . and kotlin plugin introduces two intention to do the same thing, a little bit exaggrated.